lib/hexpm/accounts/audit_log.ex

```diff
@@ -152,6 +152,9 @@ defmodule Hexpm.Accounts.AuditLog do
   defp extract_params("email.public", {old_email, new_email}),
     do: %{old_email: serialize(old_email), new_email: serialize(new_email)}
 
+  defp extract_params("email.private", {old_email, new_email}),
+    do: %{old_email: serialize(old_email), new_email: serialize(new_email)}
```

Comment:
```suggestion
  defp extract_params("email.private", {old_email, nil}),
    do: %{old_email: serialize(old_email), new_email: nil}
```

Reply:
In fact we should probably store it as:

```elixir
%{email: serialize(old_email)}
```

but I think we should leave it as is to be uniform with `email.public` log entry.